File: slo.c
//usr/bin/env gcc -o /tmp/a.out $0; /tmp/a.out; rm /tmp/a.out; exit $?
#include <stdio.h>
typedef struct Point {
  int x;
  int y;
} Point;

int main() {
  int total = 0;
  for (int i = 0; i < 1000000; i++) {
    Point p = { .x = 2, .y = 3 };
    total = total + p.x + p.y;
  }
  printf("%d\n", total);
  return 0;
}

File: slo.rs
//usr/bin/env rustc -o /tmp/a.out $0; /tmp/a.out; rm /tmp/a.out; exit $?
struct Point {
    x: i32,
    y: i32,
}

fn main() {
    let mut total = 0;
    for _ in 0..1_000_000 {
        let p = Point { x: 2, y: 3 };
        total = total + p.x + p.y;
    }
    println!("{}", total);
}
